pages/plugins/writing.md.erb

```diff
@@ -94,7 +97,20 @@ You can run the plugin linter with the following Docker command:
 docker run -it --rm -v "$PWD:/plugin:ro" buildkite/plugin-linter --id a-github-user/file-counter
 ```
 
-To make it easier to run this command, add it to the `docker-compose.yml` file:
+### Plugin linter plugin
+
+If your plugin has a Buildkite pipeline, you can add a step to lint it using the corresponding plugin:
```

Comment:
It would be good to be more specific here:

```suggestion
If your plugin has a Buildkite pipeline, you can add a step to lint it using the Plugin linter:
```

Reply:
Same here... we are talking about the buildkite plugin that runs the plugin linter (not the plugin linter itself). Yeah, I know...